.gitignore

```diff
@@ -2,52 +2,25 @@
 *.class
 *.iml
 .DS_Store
-
-# logs
 *.log
 **/core$
-
-# cache files
+hs_err_pid*
 *.cache
+*.war
+*.ear
+.clangd
 
-# MacOS dir files
-.DS_Store
-
-# IDEA config files
+# IDEA config
 .idea/
-*.iml
-*.class
-
+!.idea/vcs.xml
```

Comment:
I found this file will be updated if my local IntelliJ is upgraded. If someone updated this file, would it still work for others using old IntelliJ?

Reply:
I think in this file the structure and the components are standard. So as long as we keep using standard components, it should work across different Intellij versions.